src/app/pages/homepage/homepage.component.ts

```diff
@@ -27,4 +32,9 @@ export class HomepageComponent implements OnInit {
     this.chatMessages.push(message);
     this.chat.message = '';
   }
+  quit() {
+    this.authservice.logOutFunk();
+    console.log('Log out sucsess');
+    this.router.navigate([ '', 'land']);
```

Comment:
Incorrect logic of navigation to landing page. 
This navigation should be inside of logOutFunk() after needed actions;
Reason: logOutFunk could fail to clean data and user still will have valid tokens to work.